build.sbt

```diff
@@ -46,6 +46,25 @@ libraryDependencies += "com.datastax.cassandra" % "cassandra-driver-core" % "2.1
 
 libraryDependencies += "com.fasterxml.jackson.core" % "jackson-core" % "2.5.2"
 
+libraryDependencies ++= {
+  val sprayVersion = "1.3.3"
```

Comment:
I would move this value (and other library versions if neded be) to somethng like project/Versions.scala, then create an object Verisons and put version number there